gaphor/UML/actions/action.py

```diff
@@ -32,6 +32,64 @@ def __init__(self, diagram, id=None):
         self.watch("subject.appliedStereotype.classifier.name")
 
 
+@represents(UML.CallBehaviorAction)
+class CallBehaviorActionItem(ActionItem):
+    def __init__(self, diagram, id=None):
+        super().__init__(diagram, id)
+
+        self.shape = Box(
+            Text(
+                text=lambda: stereotypes_str(self.subject),
+            ),
+            Text(
+                text=self.name,
+                width=lambda: self.width - 4,
+            ),
+            style={
+                "padding": (4, 24, 4, 12),
+                "border-radius": 15,
+            },
+            draw=self.draw_border_with_fork,
+        )
+
+        self.watch("subject[CallBehaviorAction].behavior.name")
+
+    def name(self):
+        """Name conforms to UML2.5.1 16.3.4.1 naming description"""
+        if self.subject.behavior and not self.subject.name:
+            return self.subject.behavior.name
+        return self.subject.name or ""
```

Comment:
One thing to improve it even more is to move this code to a `format` function in `gaphor/UML/umlfmt.py`. Then the activity name will also be used as fallback in the model browser.

Just noticed this feature is not mentioned in the documentation :see_no_evil:

Reply:
I just noticed that there is a slight conflict with the name formatting when looking at SysMLv1.6. SysML requires activity name, and optionally you can also include the action name. How should we proceed?

![image](https://github.com/gaphor/gaphor/assets/45357878/975deeba-be0d-43f4-8fd5-15e30930dd52)

Reply:
We can keep is as is for now.